core/src/main/java/com/sequenceiq/cloudbreak/conf/ExternalDatabaseConfig.java

```diff
@@ -0,0 +1,97 @@
+package com.sequenceiq.cloudbreak.conf;
+
+import java.io.IOException;
+import java.util.HashSet;
+import java.util.Locale;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+
+import javax.annotation.PostConstruct;
+import javax.inject.Inject;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import com.google.common.collect.ImmutableMap;
+import com.sequenceiq.cloudbreak.common.json.JsonUtil;
+import com.sequenceiq.cloudbreak.common.mappable.CloudPlatform;
+import com.sequenceiq.cloudbreak.service.externaldatabase.DatabaseServerParameterSetter;
+import com.sequenceiq.cloudbreak.service.externaldatabase.model.DatabaseStackConfig;
+import com.sequenceiq.cloudbreak.util.FileReaderUtils;
+
+@Configuration
+public class ExternalDatabaseConfig {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(ExternalDatabaseConfig.class);
+
+    @Value("${datalake.supported.externaldb.platform:AWS}")
```

Comment:
is this a tottaly new class for datalake db ?

Reply:
the value coordinate is a copy-paste mistake. changed to a more meaningful attribute.